float64 x
float64 y
float64 theta
---
std_msgs/Empty res